csharp-basic-topics/Zip-files-in-NET/.editorconfig

```diff
@@ -0,0 +1,4 @@
+[*.{cs,vb}]

```

Comment:
We shouldn't commit this file. Please remove.

Reply:
This file is here, so the code is correctly formatted to 4 spaces. That was the requirement from Marinko in the last article, and this file is also in the git of the previous article